auth0/tenant.yaml

```diff
@@ -322,7 +330,7 @@ clientGrants:
           - 'create:interviews'
           - 'delete:interviews'
     - client_id: Compe+ API Application
-      audience: '##AUTH0_MANAGEMENT_API_AUDIENCE##'
+      audience: 'https://compe-plus-dev.us.auth0.com/api/v2/'
```

Comment:
I don't know if this should make it in?

Reply:
Actually yes good catch, otherwise we'd have a circular dependency of creating the API first or running the script first